### ISSUE TITLE
Outdated documentation

### ISSUE THREAD

**Issue by ncanning** [monetate/koupler#34]
I'm trying to build/run koupler, and following the steps from the readme, I ran
`gradle clean build copyRuntimeLibs`, which succeeded, but running `./koupler.sh` fails, due to a missing lib directory:

```
~/g/k/sh ❯❯❯ ./koupler.sh
ls: ./lib/*.jar: No such file or directory
Error: Could not find or load main class com.monetate.koupler.Koupler
```

How should I go about running this? Is the script outdated, or is some setup failing to create a lib directory? Any help would be appreciated.

**Comment by boneill42** [comment]
The shell script is meant to be run from the distribution artifact:

```
gradle clean dist
unzip build/distributions/koupler-0.2.7-SNAPSHOT.zip
cd koupler-0.2.7-SNAPSHOT/
./koupler.sh
```

I'll update the docs and potentially adjust the shell script a bit.

**Comment by boneill42** [comment]
Addressed by #32
